src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/OutputProducer.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Http
```

Comment:
Can we fix the Core project so the namespaces match the folder name? E.g. Microsoft.AspNetCore.Server.Kestrel.**Core**.Internal.Http